L/libcgal_julia_jll/Compat.toml

```diff
@@ -4,18 +4,19 @@ julia = "1"
 ["0.10-0.11"]
 libcxxwrap_julia_jll = "0.7.1"
 
+["0.16-0"]
+CGAL_jll = "5.2.0-5.2"
```

Comment:
Just to make sure there is nothing wrong: why don't you specify `CGAL_jll` for v0.10-0.11?  Apart from that, this looks ok to me

Reply:
I'll have to recall which versions they were compatible too.  I was just afraid I'd tamper with more than I needed to.  I'll get back to this ASAP.

Reply:
Pushed a change that added a compat entry for `CGAL_jll` for version up to 0.15 according to deployments to JuliaBinaryWrappers.